syntax/std_seq_array_helper.go

```diff
@@ -4,58 +4,58 @@ import (
 	"github.com/arr-ai/arrai/rel"
 )
 
-// arrayContains check if array a contains b, and b can be rel.Value or rel.Array.
-func arrayContains(a rel.Array, b rel.Value) rel.Value {
-	bArray := convert2Array(b)
-	return rel.NewBool(indexSubArray(a.Values(), bArray.Values()) > -1)
+// Checks if array subject contains sub.
+func arrayContain(subject rel.Array, sub rel.Value) rel.Value {
```

Comment:
Arg order should match the arr.ai function to avoid confusion. Check all other functions too.

Reply:
Fixed.